SecurityPkg/Library/Tpm2HelpLib/Tpm2Help.c

```diff
@@ -371,8 +367,6 @@ CopyDigestListToBuffer (
   return Buffer;
 }
 
-// MU_CHANGE - START - [TCBZ3267] Additional helper functions.

```

Comment:
Same with these MU_CHANGEs. If they are just additional helper lib functions, great, they could go into edk2 with this (as separate commits), e.g. Add Helper Lib, New Functionality to Helper Lib